test_pytest/test_c_contiguous.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
```

Comment:
should we keep directory structure same as the library (`splinepy/utils`, `splinepy/helpme`, ...)

Reply:
done